doc/users_guide/command_interface.md

```diff
@@ -0,0 +1,570 @@
+# Controlling ratpac-two 
+
+This section of the guide will provide a high-level overview of controlling ratpac-two, which is accomplished through the use of macro files.
+Macro files are plain text files containing a sequence of commands that control ratpac-two.
+Using macros, you can configure various aspects of a ratpac experiment, such as:
+
+* The physics processes to be simulated and reporting of particle track information.
+* Detector geometry, materials, and calibration constants (via RATDB).
+* Data processing chain (processors for digitization, reconstruction, output, etc.).
+* Primary particle sources (event generators).
+* Run-specific parameters like the number of events and random number seeds.
+
+Macro files are structured to follow your ratpac-two workflow.
+This is typically a **pipeline** connecting event **producers** to a sequence of **processors**.
+Each producer creates or loads an event and then passes it through the processors.
+Each processor can modify the event, record information, or simply observe it, and the order in which you declare them determines how every event will be handled.
+Ignoring the finer details of event pileup scenarios, the conceptual operation of ratpac-two looks like this:
+
+```
+Configuration of ratpac parameters (RATDB, etc.)
+
+Producer 1:
+Event 1 -> Processor 1 -> Processor 2 -> ... -> Processor N
+Event 2 -> Processor 1 -> Processor 2 -> ... -> Processor N
+...
+Event M ->  Processor 1 -> Processor 2 -> ... -> Processor N
+
+Producer 2:
+Event 1 -> Processor 1 -> Processor 2 -> ... -> Processor N
+...
+```
+
+First, we will briefly review ratpac-two commands and their syntax.
+Then we will work through an example synthesizing multiple commands into a macro file. 
+A comprehensive understanding requires extensive knowledge of the software.
+Revisiting this guide is recommended as you read through the rest of the documentation.
+
+# Table of Contents

```

Comment:
A table of contents, with links, is naturally provided by index.rst.  I think you can remove this entire section.